tests/sagemaker/mock/__init__.py

```diff
@@ -189,11 +189,15 @@ def list_tags(self):
         Handler for the SageMaker "ListTags" API call documented here:
         https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_ListTags.html
         """
-        model_arn = self.request_params["ResourceArn"]
-
+        arn = self.request_params["ResourceArn"]
+        if 'model' in arn:
+            sagemaker_resource = 'models'
+        elif 'endpoint' in arn:
+            sagemaker_resource = 'endpoints'
         results = self.sagemaker_backend.list_tags(
-            resource_arn=model_arn,
+            resource_arn=arn,
             region_name=self.region,
+            resource_type=sagemaker_resource
```

Comment:
1. Will this work for endpoint configurations?
2. This probably won't work if an endpoint has **model** in its name

Reply:
This is strictly used for testing purposes. It could be used for endpoint configs, would have to add an extra if case.